lib/bamboo/adapters/mandrill_helper.ex

```diff
@@ -48,4 +48,20 @@ defmodule Bamboo.MandrillHelper do
   def tag(email, tags) do
     put_param(email, "tags", List.wrap(tags))
   end
+
+  @doc """
+  Setup Mandrill to send using a named template with template content. Use this
+  in conjuction with merge vars to offload template rendering to Mandrill. The
+  template name specified here must match the template name stored in Mandrill.
+
+  ## Example
```

Comment:
What do you think of adding a link in between the example and the description that links to the Mandrill page about templates?